server/logger.js

```diff
@@ -0,0 +1,15 @@
+const winston = require("winston");
+
+const logger = winston.createLogger({
+  transports: [
+    new winston.transports.Console({
+      timestamp: true,
+      format: winston.format.combine(
+        winston.format.colorize(),
+        winston.format.simple()
+      )
+    })
+  ]
+});
+
+module.exports = logger;
```

Comment:
Kjenner ikke winston, er det formattering og slikt? Jeg tror også vi må bruke frontend-logger for å få console.log til å plukkes opp i nais-containeren og sendes til kibana.

Reply:
Whoops, ja, tror dette var noe jeg testet som ble med..